src/Microsoft.DotNet.Wpf/src/WindowsBase/System/Windows/Threading/DispatcherTimer.cs

```diff
@@ -267,8 +267,8 @@ private void Restart()
                 // BeginInvoke a new operation.
                 _operation = _dispatcher.BeginInvoke(
                     DispatcherPriority.Inactive,
-                    new DispatcherOperationCallback(FireTick),
-                    null);
+                    (DispatcherOperationCallback)(state => ((DispatcherTimer)state).FireTick()),
```

Comment:
(static state => ((DispatcherTimer)state).FireTick())

Reply:
The project is being built with an older LangVersion that doesn't support using static lambdas.

Reply:
@stephentoub   Can the compiler cache it ?

Reply:
Yes,  that's the purpose of this PR. The static keyword here would have zero impact on code gen; it would purely be there to trigger errors if the code were changed to try to capture something.